src/main/scala/cromwell/engine/backend/BackendCall.scala

```diff
@@ -63,9 +63,14 @@ final case class SuccessfulExecutionHandle(outputs: CallOutputs, events: Seq[Exe
   override val result = SuccessfulBackendCallExecution(outputs, events, returnCode, hash, resultsClonedFrom)
 }
 
-final case class FailedExecutionHandle(throwable: Throwable, returnCode: Option[Int] = None) extends ExecutionHandle {
+final case class FailedExecutionHandle(throwable: Throwable, returnCode: Option[Int] = None, events: Seq[ExecutionEventEntry] = Seq.empty) extends ExecutionHandle {
```

Comment:
it'd be really cool if we added a `Preempted Run 1 Started` and `Preempted Run 2 Preempted` to the events list when we get preempted. So the preemption attempts also show up on the timing diagrams...

Maybe a separate ticket?